makeflow/src/makeflow_wrapper_umbrella.c

```diff
@@ -145,7 +145,7 @@ void makeflow_wrapper_umbrella_preparation(struct makeflow_wrapper_umbrella *w,
 		// Therefore, we stop check the existence of umbrella_logfile here.
 
 		// add umbrella_logfile into the target files of a dag_node
-		if(remote_rename_support) {
+		if(remote_rename_support && !cur->local_job) {
```

Comment:
I think you are right. I did't realize `dag_node_add_target_file(cur, umbrella_logfile, umbrella_logfile);` == `dag_node_add_target_file(cur, umbrella_logfile, NULL);`.